content/intro-to-reading-data.md

```diff
@@ -59,11 +66,11 @@ Per the [JSON-RPC 2.0 specification](https://www.jsonrpc.org/specification)
 > *JSON-RPC is a stateless, light-weight remote procedure call (RPC) protocol. Primarily this specification defines several data structures and the rules around their processing. It is transport agnostic in that the concepts can be used within the same process, over sockets, over http, or in many various message passing environments. It uses [JSON](http://www.json.org/) ([RFC 4627](http://www.ietf.org/rfc/rfc4627.txt)) as data format.*
 >
 
-In practice, this specification simply involves sending a JSON object representing a method you want to call. You can do this with sockets, http, and more.
+In practice, this specification simply involves sending a JSON object representing a method you want to call. You can do this with [websocket](https://en.wikipedia.org/wiki/WebSocket), http, and more.
 
 This JSON object needs four members:
 
-- `jsonrpc` - The JSON RPC version number. This needs to be *exactly* `"2.0"`.
+- `jsonrpc` - The JSON RPC version number. This needs to be *exactly* `&quot;2.0&quot;`.
```

Comment:
We don't need HTML entities post UTF-8. Or short version: no need to change this.